src/engine/shared/config_variables.h

```diff
@@ -8,66 +8,77 @@
 #include "././game/variables.h"
 
 
-MACRO_CONFIG_UTF8STR(PlayerName, player_name, MAX_NAME_ARRAY_SIZE, MAX_NAME_LENGTH, "nameless tee", CFGFLAG_SAVE|CFGFLAG_CLIENT, "Name of the player")
```

Comment:
Avoid these format changes here, as they make is hard to review what actually changed.